### PR TITLE
fix: throw if value not a valid json

### PR DESCRIPTION
**Pull request by omerlh**

### CODE REVIEW

File: tests/specs/tests.spec.ts [hunk 1]
```diff
@@ -230,6 +230,36 @@ describe('tests', () => {
...
+        const topic = 'foo89';
+        await start([topic, deadLetterTopic], [{topic, targetPath: '/consume'}], {
+            DEAD_LETTER_TOPIC: deadLetterTopic,
+        });
+
+        const consumerMapping = await mockHttpTarget('/consume', 500);
+
+        await producer.send({topic, messages: [{value: '', key: 'thekey'}]});
+
+        // because we need Hamsa Hamsa Hamsa for tests to work
```
Comment (noamokman) [inline tests/specs/tests.spec.ts:244]: please delete the comment 🤦🏻

Comment (omerlh) [reply tests/specs/tests.spec.ts:244]: No. It will make the tests fails. I am not messing with this kind of things.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "dafka-consumer",
-    "version": "7.6.0",
+    "version": "7.6.1",
```
Comment (noamokman) [inline package.json:3]: No need to bump

Comment (omerlh) [reply package.json:3]: Are you sure? I think it is not publishing a version if this is not changed

**Review by noamokman — APPROVED** [review]

**Pull request merged**